src/message/MessageList.js

```diff
@@ -17,6 +17,11 @@ const styles = StyleSheet.create({
 export default class MessageList extends React.PureComponent {
   autoScrollToBottom = false;
 
+  constructor(props) {
+    super();
+    this.state = { actionSheetButtons: ['', ''] };
```

Comment:
Also, note that any state change does a new render. In this case, it is likely not a perf issue, but still annoys me :)